src/content/en/updates/2019/06/image-support-for-async-clipboard.md

```diff
@@ -61,13 +64,13 @@ const getClipboardContents = async () => {
 
 ### Handling paste events
 
-Paste events can be handled by using the, surprise, `"paste"` event.
+Paste events can be handled by using the, surprise, `onpaste` event.
```

Comment:
Nit: You probably have a style guide to be consistent here, but "paste" feels better to me than "onpaste". The name of the event is "paste" whereas "onpaste" is the name of the property on the DOM element that specifies the handler for "paste" events.

Reply:
Thanks, fixed.